server/repository/repository.proto

```diff
@@ -93,7 +90,14 @@ service RepositoryService {
 
 	// GetAppDetails returns application details by given path
 	rpc GetAppDetails(RepoAppDetailsQuery) returns (repository.RepoAppDetailsResponse) {
-		option (google.api.http).get = "/api/v1/repositories/{repo}/apps/{path}";
```

Comment:
is this a breaking change?

Reply:
Yes, you are right. Probably it is ok to have a breaking change in REST, since only UI is using it but I've also changed RepoAppDetailsResponse field, so gRPC is not backward compatible.

I forgot to mention it in PR. This is required to fix an old bug. If user explicitly specify app type the `GetAppDetails ` ignores it and returns details of auto-detected app type, so I had to make this change.

Increasing min api version in https://github.com/argoproj/argo-cd/blob/010dd02ba8c438b2abd5328f0e49ac9e09054cba/common/common.go#L144